<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-08T18:57:06 GST" name="Default suite" finished-at="2025-08-08T18:57:15 GST" duration-ms="8768">
    <groups>
    </groups>
    <test started-at="2025-08-08T18:57:06 GST" name="Default test" finished-at="2025-08-08T18:57:15 GST" duration-ms="8768">
      <class name="testScript.LoginTest">
        <test-method is-config="true" signature="initialiseBrowser()[pri:0, instance:testScript.LoginTest@65fb9ffc]" started-at="2025-08-08T18:57:06 GST" name="initialiseBrowser" finished-at="2025-08-08T18:57:11 GST" duration-ms="5365" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialiseBrowser -->
        <test-method signature="verifyLoginWithValidUsernameInvalidPassword()[pri:2, instance:testScript.LoginTest@65fb9ffc]" started-at="2025-08-08T18:57:11 GST" name="verifyLoginWithValidUsernameInvalidPassword" description="Verify whether user is able to login with valid username and invalid password" finished-at="2025-08-08T18:57:14 GST" duration-ms="2574" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User was able to login with invalid password expected [7rmar supermarket] but found [7rmart supermarket]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User was able to login with invalid password expected [7rmar supermarket] but found [7rmart supermarket]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testScript.LoginTest.verifyLoginWithValidUsernameInvalidPassword(LoginTest.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithValidUsernameInvalidPassword -->
        <test-method is-config="true" signature="driverQuit(org.testng.ITestResult)[pri:0, instance:testScript.LoginTest@65fb9ffc]" started-at="2025-08-08T18:57:14 GST" name="driverQuit" finished-at="2025-08-08T18:57:15 GST" duration-ms="746" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginWithValidUsernameInvalidPassword status=FAILURE method=LoginTest.verifyLoginWithValidUsernameInvalidPassword()[pri:2, instance:testScript.LoginTest@65fb9ffc] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverQuit -->
      </class> <!-- testScript.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
